orbit/CHANGELOG.md

```diff
@@ -1,21 +1,29 @@
+## Orbit 0.0.7 (Mar 8, 2022)
+
+- Improve reliability of osquery extension connection at startup.
+
+- Fix orbit not detecting updates at startup when they are published while orbit was not running.
+
+- Set log path when launching osquery.
```

Comment:
- Create and set log path for "result" and "status" logs when launching osquery.